core/dbt/graph/selector_methods.py

```diff
@@ -540,7 +540,7 @@ def search(self, included_nodes: Set[UniqueId], selector: str) -> Iterator[Uniqu
             )
 
         current_state_sources = {
-            result.unique_id: getattr(result, "max_loaded_at", None)
```

Comment:
@sungchun12 This should work right?  Mypy was complaining that line 571 is potentially an invalid statement given you can't do greater-than / less-than comparisons on `None`

Reply:
Yes this should work. As a fail-safe, I force a condition for it to have the attribute in the first place on line 545 for example

```python
if hasattr(result, "max_loaded_at")
        }
```